### PR TITLE
Mailing dependency injection

### PR DESCRIPTION
**Pull request by valeriaDD**

### CODE REVIEW

File: app/Services/ContactUsMailer.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+                        'messageText' => $data['messageText']
+                    ], 
+
+                    function (Message $message) use ($data){
+ 
+                             $message->subject('User Message ' . $data['email']);
+                             $message->to('tech@lotus.app');
+                             $message->from('dontReply@lotus.app', 'Lotus');
+        });
+        $this->logger->info('Contact us Mail: A user have sent you a message!');
```
Comment (egorbwork) [inline app/Services/ContactUsMailer.php:35]: As recommendation, indicate to which email you sent the message, could be easier to find some data.

File: app/Services/ContactUsMailer.php [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+    public function __construct(Mailer $infrastructureMailer, LoggerInterface $logger){
+         $this->infrastructureMailer = $infrastructureMailer;
+         $this->logger = $logger;
+    }
+    
+    public function send(array $data):void
+    {
+        $this->infrastructureMailer->send('Emails.ContactUs',
+                    [
```
Comment (egorbwork) [inline app/Services/ContactUsMailer.php:22]: you still should data as you have adjusted in the mailing pr.

**Review by egorbwork — CHANGES_REQUESTED** [review]
Please improve a bit current solution.

File: app/Services/ContactUsMailer.php (hunk 2 above)
Comment (valeriaDD) [reply app/Services/ContactUsMailer.php]: Fixed- excluded data fracturing as in the previous part

File: app/Services/ContactUsMailer.php (hunk 1 above)
Comment (valeriaDD) [reply app/Services/ContactUsMailer.php]: Fixed- made a more explicit logger message that includes the sender and the receiver of the email

**Review by egorbwork — APPROVED** [review]
Now looks finished. Good job!

**Pull request merged**